check/format-incremental

```diff
@@ -66,6 +66,17 @@ if [ -z "${rev}" ]; then
 fi
 echo "Comparing against revision '${rev}'." >&2
 
+# Copy over .yapfignore or use local version. Cleanup if copying.
+ignore_exists='[ -f .yapfignore ]'
+if ! $ignore_exists; then
+  function cleanup {
+    rm .yapfignore
```

Comment:
Can we pass this file in as a command line parameter, instead of by copying and deleting?

Reply:
yapf doesn't allow a config path to be passed in :(